src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -29,6 +29,26 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Property
     End Class
 
+    Partial Friend Class BoundAddressOfOperator
+        Protected Overrides ReadOnly Property Children As ImmutableArray(Of BoundNode)
+            Get
+                Return ImmutableArray.Create(Of BoundNode)(Me.MethodGroup)
+            End Get
+        End Property
+    End Class
+
+    Partial Friend Class BoundMethodGroup
```

Comment:
>BoundMethodGroup [](start = 25, length = 16)

I don't think it is useful to expose type arguments. The receiver is interesting, consider exposing it by implementing Children in BoundMethodOrPropertyGroup so that Property groups are covered as well

Reply:
I think I have to disagree here. If I were trying to implement a type parameter unused analyzer, I'd need to know if the type parameter was used in invalid scenarios like this. I will add the implementation for BoundMethodOrPropertyGroup though.

Reply:
> I think I have to disagree here. If I were trying to implement a type parameter unused analyzer, I'd need to know if the type parameter was used in invalid scenarios like this

Type arguments are always used, unless there is a binding error.